docs/workflow/pycbc_make_coinc_search_workflow.rst

```diff
@@ -608,53 +608,250 @@ If you have a subdax that failed, ``pegasus_analyzer`` will provide you with a c
-``--output`` argument to ``pycbc_make_coinc_search_workflow``. This file contains a 
-listing of all of the data products of the prior workflow, and can be used to tell
-pegasus to skip regenerating them.
-
-Select the entries in the file that you would like to skip generating again and
-place that into a new file. The example below selects all the inspiral and 
-tmpltbank jobs and places their entries into a new listing called prior_data.map.::
-
-    # Lets get the tmpltbank entries
-    cat /path/to/old/run/${GPS_START_TIME}-${GPS_END_TIME}/output.map | grep 'TMPLTBANK' > prior_data.map
-    
-    # Add in the inspiral  files
-    cat /path/to/old/run/${GPS_START_TIME}-${GPS_END_TIME}/output.map | grep 'INSPIRAL' >> prior_data.map
+One of the features of Pegasus is reuse the data products of prior runs.
+This can be used to e.g. expand an analysis or recover a run with mistaken settings without
+duplicating work. The steps below explain how to do this.
+
+------------------------------------
+Setting up a workflow for data reuse
+------------------------------------
+
+The first step is to generate a new workflow that performs the analysis that
+you would like to do. This workflow should be generated in a new directory so that it does not overwrite data from your previous workflows.
+Data reuse happens at the ``pycbc_submit_dax`` step, so
+first run ``pycbc_make_coinc_search_workflow`` to build a new workflow,
+following the instructions in the section :ref:`coincworkflowgenerate` of this
+page.
+
+**Stop** before you plan and submit the workflow with ``pycbc_submit_dax``.
+You will pass an additional file to ``pycbc_submit_dax`` using the
+``--cache-file`` option with a list of files that Pegasus can re-use from a
+previous run.  The Pegasus Workflow Planner will reduce the workflow
+using this cache file. Reduction works by deleting jobs from the workflow
+whose output files have been found in some location in this cache file.
+
+The key to data reuse is building the cache file passed to ``pycbc_submit_dax``. This file maps a file created in the workflow to a URL and a site where that URL can be found. The syntax of the cache file is plain ASCII with each line in the file giving the location of a file in the format::
+
+    LOGICAL_FILE_NAME PHYSICAL_FILE_URL pool="SITE"
+
+where ``LOGICAL_FILE_NAME`` is the name of the file as it appears in the
+workflow (incluing any preceeding path), ``PHYSICAL_FILE_URL`` is a URL where
+the file can be found, and ``SITE`` is the site on which that URL resides.
+
+The URI in the ``PHYSICAL_FILE_URL`` can be any of the URIs that Pegasus
+recognizes. The URIs ``file://``, ``gsiftp://``, and ``http://`` are likely
+the most useful. Pegasus will take care of adding transfer jobs for
+``gsiftp://`` and ``http://`` URIs, if the data is not available locally.
+
+The string ``SITE`` is a hint that tells Pegasus on which site the
+``PHYSICAL_FILE_URL`` can be found (This is needed beacuse the execution site
```

Comment:
i don't know what 'execution site' means here. either de-jargonize or omit the explanation